client/src/components/review.tsx

```diff
@@ -11,8 +11,13 @@ import {
   IconButton,
 } from "@chakra-ui/react";
 import { BiLike, BiChat, BiShare } from "react-icons/bi";
+import { Comment } from "./interfaces/Comment";
```

Comment:
```suggestion
import type { Comment } from "./interfaces/Comment";
```
Use `import type` if you are only importing type